### ISSUE TITLE
callback of hydraExpress.init is called 2 times

### ISSUE THREAD

**Issue by k33g** [pnxtech/hydra-express#29]
Hello,

I'm trying this sample, and the callback seems to be called 2 times (it displays 2 😀)

```javascript
const hydraExpress = require('fwsp-hydra-express');
const config = require('./config.json');

hydraExpress.init(config, () => {
  console.log("😀")
  const express = hydraExpress.getExpress();
  const api = express.Router();

  api.get('/', function(req, res) {
    res.send({
      msg: `hello from ${hydra.getServiceName()} - ${hydra.getInstanceID()}`
    });
  });

  hydraExpress.registerRoutes({
    '': api
  });
});
```

**Comment by cjus** [comment]
@k33g thanks for the bug report. This should be fixed in fwsp-hydra-express@0.14.5

**Comment by k33g** [comment]
@cjus thx, it works fine 👍
